conf/SiteDefs.pm

```diff
@@ -34,9 +34,9 @@ use Sys::Hostname::Long;
 
 ###############################################################################
 ## Ensembl Version and release dates (these get updated every release)
-our $ENSEMBL_VERSION        = 114;            # Ensembl release number
-our $ARCHIVE_VERSION        = 'Apr2025';     # Archive site for this version
-our $ENSEMBL_RELEASE_DATE   = 'April 2025'; # As it would appear in the copyright/footer
+our $ENSEMBL_VERSION        = 115;            # Ensembl release number
+our $ARCHIVE_VERSION        = 'Sept2025';     # Archive site for this version
```

Comment:
I think we use only three letters.

Example: https://sep2019.archive.ensembl.org/index.html